### PR TITLE
Add support for splitting cards into fragments in matrix events

### PR DESCRIPTION
**Pull request by habdelra**
This PR adds a new custom matrix message event that holds "card fragments". This allows us to split large cards into smaller fragments that are below the matrix 65KB event threshold. We then reassemble the card fragments back into a card on the matrix client. The matrix clients in question that can split and reassemble cards are the host app and the ai bot server.

### CODE REVIEW

File: packages/base/room.gts [hunk 1]
```diff
@@ -561,6 +612,20 @@ interface RoomCreateEvent extends RoomStateEvent {
...
+  };
+}
+
+interface RoomPowerLevels extends RoomStateEvent {
+  type: 'm.room.power_levels';
+  content: {
+    // TODO
+  };
+}
+
```
Comment (backspace) [inline packages/base/room.gts:628]: Are these needed because they occur in tests or something and need to be skipped over?

**Review by backspace — APPROVED** [review]
nice end-to-end test on this 🎉

**Review by jurgenwerk — APPROVED** [review]
Looking good!

File: packages/base/room.gts (hunk 1 above)
Comment (habdelra) [reply packages/base/room.gts:628]: The intent here is to provide stronger types that what matrix provides so we can keep our events well typed. But yeah, this particular event only happens in a test, and in that test we really don’t care about the deeper details for this event. The TODO here is for when we do care about the deeper details of this event (if that ever happens)

**Pull request merged**